content/docs/start/data-management/data-pipelines.md

```diff
@@ -47,6 +47,11 @@ $ tree
     ├── prepare.py
     ├── requirements.txt
     └── train.py
+
+$ git init  # if needed
+$ dvc init
```

Comment:
folks, I think it might confuse users even more ... also I it won't make it "actionable", you need a lot of steps (dvc add / import at the very least to start). The previous section serves as an actionable item for this.

Reply:
> p.s. not sure this is wanted per [#1943 (comment)](https://github.com/iterative/dvc.org/issues/1943#issuecomment-728193024):
> 
> > Doing it completely reproducible and chapters independent makes it a hands-on tutorial like (which get started is not). It means longer read time, more distractions for users who want to get the idea quick, etc.


> folks, I think it might confuse users even more ... also I it won't make it "actionable", you need a lot of steps (dvc add / import at the very least to start). The previous section serves as an actionable item for this.

So it looks like @jorgeorpinel opened the PR to generate this discussion of whether reproducibility is wanted, and @shcheklein your opinion is that it is not wanted, correct?

It makes sense, except having the section to download the code and install requirements doesn't make sense to me in that case. Why not refer back to previous sections and leave it at that?

Reply:
> of whether reproducibility is wanted, and @shcheklein your opinion is that it is not wanted, correct?

it's needed. My point that we have enough for this (maybe do some additional hidden section with precise `git commit` commands, etc). Making every section independent goes way beyond what we had in mind for the get started and will complicate things.

Also, I don't believe this should be a priority. Even if we do this at best we'll get some small improvement. It's not worth it. We should be doing major updates (whatever it means).

Reply:
> So it looks like @jorgeorpinel opened the PR to generate this discussion of whether reproducibility is wanted

it was not clear from the description btw, so I was reading this as an improvement/bug fix PR:

<img width="327" alt="Screen Shot 2022-09-27 at 1 08 27 PM" src="https://user-images.githubusercontent.com/3659196/192625351-ec9c73d8-3406-4b78-a3fd-903f479cc43b.png">

Reply:
Opened as a quick way to make sure #3998 was closed properly (fully done) as I groomed GS-related tickets. This is definitely not high priority.

Removed `bug` label and updated description. Thanks for the other Feedback as well! Will address when possible